packages/effect/src/Number.ts

```diff
@@ -5,7 +5,7 @@
  *
  * @since 2.0.0
  */
-import * as equivalence from "./Equivalence.js"
+import * as Number_ from "./Equivalence.js"
```

Comment:
Why are we changing this import?

Reply:
I changed number variable in Number.ts to number_ for using Number.MAX_SAFE_INTEGER and Number.MIN_SAFE_INTEGER

Reply:
sorry. it seems my mistake